contents/handbook/engineering/developing-locally.md

```diff
@@ -201,11 +201,16 @@ Assuming Node.js is installed, run `yarn --cwd plugin-server` to install all req
 
 1. Install requirements with pip
 
+    Upgrade pip to the last version
+    ```
+    pip install -U pip
+    ```
```

Comment:
For clarity, this could be a separate step between "1. Install requirements with pip" and  "1. Activate the virtual environment"

Reply:
Updated ✅